packages/renderer/src/lib/preferences/PreferencesRenderingItemFormat.svelte

```diff
@@ -129,7 +129,7 @@ async function selectFilePath() {
         required />
     {:else if record.type === 'string' && record.enum && record.enum.length > 0}
       <select
-        class="border  text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 bg-gray-600 border-gray-500 placeholder-gray-400 text-white"
+        class="border-b text-md focus:ring-blue-500 focus:border-blue-500 block w-full p-1 bg-zinc-700 border-gray-500 placeholder-gray-400 text-white"
```

Comment:
I think it should be something separated (PR/commit) as it's not directly related to the PR

Reply:
It is, unfortunately without these changes, the drop-down menu doesn't look very good / not the correct colour as the other options in the menu.

Reply:
can be a separate PR being a pre-req of this one